examples/trace/client.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+#
+# Copyright 2019, OpenTelemetry Authors
```

Comment:
I wonder if we really need these copyright info in such a simple example?
These examples are not going to shipped in any final package, probably we should just treat them as markdown files (and we don't put copyright or Apache2 license in markdown files).

Reply:
That's a question for the lawyers, it may be that we don't need to include this anywhere and we're just following apache's recommendations here.

We may also be able to get away with the shorter boilerplate at http://www.apache.org/foundation/license-faq.html#Apply-My-Software.